packages/processor/processor.0.2/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+synopsis: "Processor Topology & Affinity for ocaml"
+description:
+  "Exposes processor, CPU Topology and lets you pin the runtime to a set of CPUs."
+maintainer: ["Léo Andrès <contact@ndrs.fr>"]
+authors: ["Christiano Haesbaert"]
+license: "ISC"
+tags: [
+  "processor"
+  "multicore"
+  "cpu"
+  "thread"
+  "core"
+  "package"
+  "socket"
+  "affinity"
+  "scheduler"
+  "domain"
+]
+homepage: "https://github.com/haesbaert/ocaml-processor"
+doc: "https://haesbaert.github.io/ocaml-processor"
+bug-reports: "https://github.com/haesbaert/ocaml-processor/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "3.2"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/haesbaert/ocaml-processor.git"
+url {
+  src:
+    "https://github.com/haesbaert/ocaml-processor/archive/refs/tags/0.2.tar.gz"
+  checksum: [
+    "sha256=35df0b95a8d4b420feea0ec98bd3b83592884764a586643979098bdbf3fd5ba3"
+    "sha512=c9ae036fa90009352775d61d6ae7cfdbe9535677a59c0cab4a6ea6a26e63fc775d48606e7a0c55ba4486f42021b4979cd3eb5f68d9366d0cda8cb9067a531016"
+  ]
+}
```

Comment:
```suggestion
}
available: os-family != windows & (ocaml-version >= "5.0" | (arch != "arm32" & arch != "ppc64" & arch != "s390x" & arch != "x86_32"))
```

(There might be a typo in there, please re-read before applying?)

Reply:
I'm not sure I get why you added `ocaml-version >= "5.0"`, could you explain?

Reply:
The CI seem to pass on ocaml>=5.0. E.g.,


> x86_32-ocaml-4.14
> [processor.0.2 (failed: Some modules don't have an implementation.)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/bbb19f34aaeaef7a8912e22ecc6606de1d28a98e/variant/extras,x86_32-ocaml-4.14,processor.0.2)
> x86_32-ocaml-5.2
> [processor.0.2 (passed)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/bbb19f34aaeaef7a8912e22ecc6606de1d28a98e/variant/extras,x86_32-ocaml-5.2,processor.0.2)
> [tests (passed)](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/bbb19f34aaeaef7a8912e22ecc6606de1d28a98e/variant/extras,x86_32-ocaml-5.2,processor.0.2,tests)

I'm not sure if the CI is a reasonable approximation of the actual success/failure (it's possible this is a false negative)… What do you think?

Reply:
Support for i386 was removed in OCaml 5, how can the CI run 5.2 on it ?

Reply:
For some of the ocaml-version×acrhitecture, the code is only tested in bytecode. I think that's what's going on here.

Btw, if you want to avoid the bytecode-only combinations, you can add `conflicts: [ "ocaml-option-bytecode-only" ]` to your opam file.